controllers/user.js

```diff
@@ -234,8 +234,29 @@ exports.userListPage = function (aReq, aRes, aNext) {
       pageMetadata(options, ['Flagged Users', 'Moderation']);
     }
   }
-  function render() { aRes.render('pages/userListPage', options); }
-  function asyncComplete(err) { if (err) { return aNext(); } else { preRender(); render(); } }
+  function render() {
+    aRes.render('pages/userListPage', options);
+  }
+  function asyncComplete(aErr) {
+    if (aErr) {
+      aNext();
+      return;
+    }
```

Comment:
Kind of figured that with the missing `aNext()` and it raised an eyebrow yesterday in my background thinking... I think I'll do that in a single PR with some cleanup just on `./controllers/user.js`... I read elsewhere that ES6 is going to need `undefined` for certain functions instead of `null` *(or worse some other huge object in a bug)* so that is why I have been splitting out the `aNext()` and `return` statements... also it makes more sense not to return something that isn't there and could change down the line.

Just a misc note too *node* 4.x went LTS according to their site e.g. 5.x is around some corner.